TUM Campus App/AppDelegate.swift

```diff
@@ -15,42 +15,26 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        //UIApplication.sharedApplication().setStatusBarStyle(UIStatusBarStyle.LightContent, animated: true)
+        setupAppearance()
+        conditionallyShowLoginViewController()
+
+        return true
+    }
+
+    private func setupAppearance(){
```

Comment:
Please fix the spacing here as well. I know it's tedious, but my OCD and my internal linter are going crazy when I see this